api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java

```diff
@@ -70,11 +70,13 @@ public Person getPersonByUuid(String uuid) {
 			criteria.createAlias("addresses", "pad");
 			criteria.add(c);
 		});
-		if (!containsAlias(criteria, "pn")) {
-			criteria.createAlias("names", "pn");
-		}
-		if (!containsAlias(criteria, "pad")) {
-			criteria.createAlias("addresses", "pad");
+		if (sort != null) {
```

Comment:
I still don't like this solution. For example, if I were to sort by "birthdate", I'd still get joins against `PersonName` and `PersonAddress`.

Reply:
But the `sort.getParamName().equals(...)` condition will return false then, right?

Reply:
Whoops! I missed that! Sorry, too much time staring at git diffs.

Reply:
It's alright, happens to me all the time! 😄